src/changelog.ts

```diff
@@ -264,28 +354,42 @@ export default class Changelog {
 
   #formatter: Formatter;
 
+  readonly #versionBeforePkgRename: string | undefined;
+
+  readonly #tagPrefixBeforePkgRename: string | undefined;
+
```

Comment:
With `exactOptionalPropertyTypes` disabled, the `| undefined` is redundant.
```suggestion
  readonly #versionBeforePkgRename?: string;

  readonly #tagPrefixBeforePkgRename?: string;
```